diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt

```diff
@@ -491,35 +494,98 @@ class NewlinesRule(configRules: List<RulesConfig>) : DiktatRule(
             }
         }
 
-    private fun handleValueParameterList(node: ASTNode, entryType: String) = node
-        .children()
-        .filter {
-            val isNewLineNext = it.treeNext?.isNewLineNode() ?: false
-            val isNewLinePrev = it.treePrev?.isNewLineNode() ?: false
-            (it.elementType == COMMA && !isNewLineNext) ||
-                    // Move RPAR to the new line
-                    (it.elementType == RPAR && it.treePrev?.elementType != COMMA && !isNewLinePrev)
+    private fun isCorrectSuperTypeList(valueParameterList: List<ASTNode>): Boolean {
```

Comment:
please add kdoc, where you describe about what is **correct** supertype list

Reply:
Done